gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-all.zip
```

Comment:
Should we update the `minimumVersion` in `settings.gradle`? (some changes in `7.2` seem backwards incompatible)

```
def minimumVersion = GradleVersion.version("3.5")
if (gradleVer < minimumVersion) {
  throw new GradleScriptException("this build cannot be run with gradle < " + minimumVersion + ". current detection version is " +
      gradleVer + ". use newer gradle or (better yet) use the wrapper", null)
}
```

Reply:
Done